packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -221,7 +223,21 @@ function Cesium3DTileset(options) {
     options.maximumScreenSpaceError,
     16
   );
-  this._maximumMemoryUsage = defaultValue(options.maximumMemoryUsage, 512);
+  this._memoryAdjustedScreenSpaceError = this._maximumScreenSpaceError;
+
+  let defaultCacheBytes = 512 * 1024 * 1024;
+  if (defined(options.maximumMemoryUsage)) {
+    deprecationWarning(
+      "Cesium3DTileset.maximumMemoryUsage",
+      "Cesium3DTileset.maximumMemoryUsage was deprecated in CesiumJS 1.106.  It will be removed in CesiumJS 1.109. Use Cesium3DTileset.cacheBytes instead."
+    );
+    defaultCacheBytes = options.maximumMemoryUsage * 1024 * 1024;
+  }
+  this._cacheBytes = defaultValue(options.cacheBytes, defaultCacheBytes);
+  this._cacheHeadroomBytes = Math.max(
```

Comment:
I would use `Check.typeOf.number.greaterThan` instead of silently overriding the provided value.